crowbar_framework/app/models/node_object.rb

```diff
@@ -1360,7 +1360,7 @@ def disk_release(device, owner)
     if device
       crowbar_wall[:claimed_disks] ||= {}
 
-      unless disk_owner(device) == owner
+      if owner.empty? || disk_owner(device) != owner
```

Comment:
It was failing without this added test, so maybe there was only a check for nil before